### PR TITLE
Use factories in cli commands to prevent installation error

### PR DESCRIPTION
**Pull request by pmclain**
Removes dependencies that require installation before instantiation. This was preventing CLI installation, which meant no travis builds :(

### CODE REVIEW

File: Command/Runjob.php [hunk 1]
```diff
@@ -5,38 +5,39 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Input\InputArgument;
+use EthanYehuda\CronjobManager\Model\ManagerFactory;
 use EthanYehuda\CronjobManager\Model\Manager;
```
Comment (Ethan3600) [inline Command/Runjob.php]: If we're using the factory, we don't need to use `use EthanYehuda\CronjobManager\Model\Manager;` anymore

File: Command/Showjobs.php [hunk 2]
```diff
@@ -4,16 +4,17 @@
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use EthanYehuda\CronjobManager\Model\ManagerFactory;
 use EthanYehuda\CronjobManager\Model\Manager;
```
Comment (Ethan3600) [inline Command/Showjobs.php]: Same thing here, I don't think we need this anymore

**Review by Ethan3600 — CHANGES_REQUESTED** [review]
Just some namespaces no longer needed

**Comment by pmclain** [comment]
@Ethan3600 agreed, removed.

**Review by Ethan3600 — APPROVED** [review]

**Pull request merged**